web/src/click_handlers.ts

```diff
@@ -462,6 +462,12 @@ export function initialize(): void {
         if ($(e.target).parents(".user-profile-picture").length === 1) {
             return;
         }
+        if (document.getSelection()?.type === "Range") {
+            e.preventDefault();
+            // To prevent blur caused by other link click handlers.
+            e.stopPropagation();
+            return;
+        }
```

Comment:
Hmm, which specific handler is at fault here? I wonder if that click handler needs the `Range` check instead of the `stopPropagation` here.